docs/percona-for-mysql.md

```diff
@@ -12,17 +12,18 @@ hide:
 
 # Percona Software for MySQL
 
-Quickstart guides, user guides, and how-tos, release notes, and the latest downloads. Everything you need to optimize your MySQL databases is right here.
+Quickstart guides, user guides, and how-tos, release notes, and the latest downloads. Everything you need to optimize your MySQL databases is right here
```

Comment:
remove the first "and"
Quickstart guides, user guides, how-tos, release notes,

What does "and the latest downloads" mean?

Reply:
updated